core/interop/source-generation/ComWrappersGeneration/Shared/ICalculator.cs

```diff
@@ -0,0 +1,14 @@
+using System.Runtime.InteropServices;
+using System.Runtime.InteropServices.Marshalling;
+
+namespace Tutorial;
+
+[GeneratedComInterface]
+[InterfaceType(ComInterfaceType.InterfaceIsIUnknown)]
```

Comment:
It's not required, but I thought it would be good to leave to make it clear that the interface derives only from IUnknown, and in case we start supporting IDispatch.

Reply:
Okay. Doesn't seem too unreasonable.